samples/filterSnippets.js

```diff
@@ -0,0 +1,351 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// sample-metadata:
+//   title: Read a row
+//   description: Get all the data for a Bigtable row by key.
+//   usage: node readRow.js <instanceId>
+
+function main(
+  instanceId = 'YOUR_INSTANCE_ID',
+  tableId = 'YOUR_TABLE_ID',
+  filterType = 'filterRowSample'
+) {
+  // [START bigtable_filters_limit_row_sample]
+  // [START bigtable_filters_limit_row_regex]
+  // [START bigtable_filters_limit_cells_per_col]
+  // [START bigtable_filters_limit_cells_per_row]
+  // [START bigtable_filters_limit_cells_per_row_offset]
+  // [START bigtable_filters_limit_col_family_regex]
+  // [START bigtable_filters_limit_col_qualifier_regex]
+  // [START bigtable_filters_limit_col_range]
+  // [START bigtable_filters_limit_value_range]
+  // [START bigtable_filters_limit_value_regex]
+  // [START bigtable_filters_limit_timestamp_range]
+  // [START bigtable_filters_limit_block_all]
+  // [START bigtable_filters_limit_pass_all]
+  // [START bigtable_filters_modify_strip_value]
+  // [START bigtable_filters_modify_apply_label]
+  // [START bigtable_filters_composing_chain]
+  // [START bigtable_filters_composing_interleave]
+  // [START bigtable_filters_composing_condition]
+  const Bigtable = require('@google-cloud/bigtable');
+  const bigtable = Bigtable();
+
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const instanceId = 'YOUR_INSTANCE_ID';
+  // const tableId = 'YOUR_TABLE_ID';
+  const instance = bigtable.instance(instanceId);
+  const table = instance.table(tableId);
+
+  // [END bigtable_filters_limit_row_sample]
+  // [END bigtable_filters_limit_row_regex]
+  // [END bigtable_filters_limit_cells_per_col]
+  // [END bigtable_filters_limit_cells_per_row]
+  // [END bigtable_filters_limit_cells_per_row_offset]
+  // [END bigtable_filters_limit_col_family_regex]
+  // [END bigtable_filters_limit_col_qualifier_regex]
+  // [END bigtable_filters_limit_col_range]
+  // [END bigtable_filters_limit_value_range]
+  // [END bigtable_filters_limit_value_regex]
+  // [END bigtable_filters_limit_timestamp_range]
+  // [END bigtable_filters_limit_block_all]
+  // [END bigtable_filters_limit_pass_all]
+  // [END bigtable_filters_modify_strip_value]
+  // [END bigtable_filters_modify_apply_label]
+  // [END bigtable_filters_composing_chain]
+  // [END bigtable_filters_composing_interleave]
+  // [END bigtable_filters_composing_condition]
+
+  // [START bigtable_filters_limit_row_sample]
+  async function filterRowSample() {
+    const filter = {
+      row: {
+        sample: 0.75,
+      },
+    };
+    readWithFilter(filter);
+  }
+  // [END bigtable_filters_limit_row_sample]
+  // [START bigtable_filters_limit_row_regex]
+  async function filterRowRegex() {
```

Comment:
Here and below - no need for these methods to be async unless they do an `await` inside.